lms/templates/static_pdfbook.html

```diff
@@ -48,7 +48,7 @@
   height="1108"
   frameborder="0"
   seamless
-  tabindex="1"
+  tabindex="0"
```

Comment:
@ssemenova I think we can remove the tabindex value here all together.  While you are in here, please remove the aria-live="polite" all together there.  That definitely doesn't belong there.  thanks!